commit 02ae35c9e105
Author: Botond Ballo <botond@mozilla.com>
Date:   Wed Oct 18 13:29:47 2017 -0400

    Bug 1400238 - Notify TabParent when the GTK client offset changes. r=karlt
    
    MozReview-Commit-ID: 1TacAhl6c2M
    
    --HG--
    extra : rebase_source : 5641d417a6245cc2f8027fd91032119f1b36d461
---
 widget/gtk/nsWindow.cpp | 5 +++++
 1 file changed, 5 insertions(+)

diff --git widget/gtk/nsWindow.cpp widget/gtk/nsWindow.cpp
index 955327297ac9..e58f3dcb59d5 100644
--- widget/gtk/nsWindow.cpp
+++ widget/gtk/nsWindow.cpp
@@ -1567,6 +1567,11 @@ nsWindow::OnPropertyNotifyEvent(GtkWidget* aWidget, GdkEventProperty* aEvent)
 {
   if (aEvent->atom == gdk_atom_intern("_NET_FRAME_EXTENTS", FALSE)) {
     UpdateClientOffset();
+
+    // Send a WindowMoved notification. This ensures that TabParent
+    // picks up the new client offset and sends it to the child process
+    // if appropriate.
+    NotifyWindowMoved(mBounds.x, mBounds.y);
     return FALSE;
   }
 
